packages/workers/importer/protocol.ts

```diff
@@ -80,11 +81,40 @@ export type ImporterApi = Api<typeof protocol>;
 export class ImporterConfig {
   static Codec = codec.Class(ImporterConfig, {
     omitSealVerification: codec.bool,
+    pvm: codec.u8.convert(
+      (i) => {
+        switch (i) {
+          case PVMBackend.BuildIn:
```

Comment:
Instead of complicating the codec, you might rather want to use number instead of a string in enum definition and create a function that turns that number/enum into `string`.